app/controllers/application_controller.rb

```diff
@@ -1,7 +1,15 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+  end
+
   def current_user
-    @current_user = User.find(1)
+    # @current_user = User.find(1)
   end
```

Comment:
- [ ] I suggest that you remove this `current_user` method in ApplicationController since device will provide us with one.

Reply:
done!